httpclient/src/main/java/org/apache/http/impl/auth/GGSSchemeBase.java

```diff
@@ -66,11 +66,22 @@
         FAILED,
     }
 
+    /** Kerberos flag options */
+    enum Option
+    {
+        DEFAULT,
+        ENABLE,
+        DISABLE
+    }
+
+    private static Option defaultRequestDelegCreds = Option.DEFAULT;
```

Comment:
Do we really need a static _mutable_ variable? I think it is completely unnecessary

Reply:
It's because I meant it to be a configurable 'default/global' behavior.  For example, in our case, we'll want to set this to ENABLED.  In so doing, when we create an HttpClient using the builder, or when 3rd party libraries create HttpClient objects internally, that they be instantiated the way they need to to (i.e.: with delegation working).

Reply:
But likewise flipping a global flag on an unsuspected 3rd party library can break things in non-obvious ways. I personally think that global mutable variables is a very bad idea.